src/logger/src/logger.rs

```diff
@@ -475,24 +468,20 @@ impl fmt::Display for LoggerError {
 
 /// Implements the "Log" trait from the externally used "log" crate.
 impl Log for Logger {
-    // Test whether the level of the log line should be outputted or not based on the currently
-    // configured level. If the configured level is "warning" but the line is logged through "info!"
-    // marco then it will not get logged.
-    fn enabled(&self, metadata: &Metadata) -> bool {
-        metadata.level() as usize <= self.level.load(Ordering::SeqCst)
+    // This is currently not used.
+    fn enabled(&self, _metadata: &Metadata) -> bool {
```

Comment:
nit: Why not delete it?

Reply:
It's part of the `Log` trait and all the implementers have to define this method. Otherwise the compiler throws a `not all trait items implemented` error.

Reply:
Sorry, can you expand on that please ? I'm not sure I understand how.

Reply:
My comment was more of an FYI and it is related to the other issue actually: https://github.com/firecracker-microvm/firecracker/issues/1686.

Let's sync on this offline as it is a lot to write in a comment.